.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 sudo: required
 before_install:
-- echo $GCLOUD_CREDENTIALS | openssl enc -d -aes-256-cbc -base64 -A -k $GCLOUD_CREDENTIALS_KEY > fhir-org-cds-services.json
+- if [[ ! -z "$GCLOUD_CREDENTIALS" ]]; then echo $GCLOUD_CREDENTIALS | openssl enc -d -aes-256-cbc -base64 -A -k $GCLOUD_CREDENTIALS_KEY > fhir-org-cds-services.json; fi
```

Comment:
Because this PR is from a fork and travis-ci [blocks access to environment variables](https://docs.travis-ci.com/user/pull-requests/#pull-requests-and-security-restrictions) for PRs from forks, initially, the [build failed](https://travis-ci.org/cds-hooks/sandbox-cds-services/builds/539841342).

I added this bash to allow the PR build to continue for forks, which allowed a successful build - https://travis-ci.org/cds-hooks/sandbox-cds-services/builds/539861548

Once the PR is merged, we'll have to ensure that the deployment itself is still successful.